spec/models/conference_spec.rb

```diff
@@ -510,13 +510,13 @@
   describe 'event type distribution' do
     before do
       subject.email_settings = create(:email_settings)
-      @workshop = create(:event_type, title: 'Workshop', color: '#000000')
-      @lecture = create(:event_type, title: 'Lecture', color: '#ffffff')
+      @workshop = create(:event_type, title: 'Workshop', color: '#000000', conference: subject)
+      @lecture = create(:event_type, title: 'Lecture', color: '#ffffff', conference: subject)
     end
 
     describe '#event_type_distribution' do
       it 'calculates correct for different event types' do
-        create(:event, conference: subject, event_type: @workshop)
+        blah = create(:event, conference: subject, event_type: @workshop)
```

Comment:
This was only for debugging, we should remove it.
(Rubocop also reports about it!)